src/index.html

```diff
@@ -5,8 +5,409 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+
+    <link
+    rel="preconnect"
+    href="https://fonts.googleapis.com"
+    >
+
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    >
+
+    <link 
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a 
+        href="#home"
+        class="logo">
```

Comment:
check your code style. not necessary to move to a new line if you have only two attributes

Reply:
anyway closing bracket should be on a new line